.gitignore

```diff
@@ -0,0 +1 @@
+greeting.js
```

Comment:
Did you add this because [this line](https://github.com/nodeshift/ubi8-s2i-nodejs/pull/1/files#diff-a8b82b354f2e547fabfa7df91cc11b8fR9) generates the file?

Reply:
I was added because it was in https://github.com/nodeshift/centos7-s2i-nodejs/blob/master/.gitignore. Note sure it is there.